exercises/elm-1-a/README.md

```diff
@@ -0,0 +1,17 @@
+## Introduction
+
+This is Part 1 of our research into how people write Elm differently. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+Your task is to take a list of scorers for a football game, and aggregate any duplicates. The aggregated list should represent each scorer once with the number of goals after their name in brackets, if they have scored multiple goals.
+
+The aggregated list should be ordered alphabetically.
+
+For example, if the initial list of scorers is
+
+["Lisa", "Anna", "Lisa"]
```

Comment:
A small suggestion could be to show an actual function call here:

```elm
aggregateScorers ["Lisa", "Anna", "Lisa"]
```

You could also show the expected result:

```elm
aggregateScorers ["Lisa", "Anna", "Lisa"]
-- Returns: [ "Anna", "Lisa (2)" ] 
```